### PR TITLE
magento/devdocs#: Resolvers. Deprecated option.

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this PR

This PR extends list of resolver mutation syntax. Add information about `@deprecated` directive.

## Affected URLs

- https://devdocs.magento.com/guides/v2.3/graphql/develop/resolvers.html

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Codebase: https://github.com/webonyx/graphql-php/blob/11d32d456848a554e7615c3e21d18fa0c10152d5/src/Utils/SchemaPrinter.php#L402

(Webonyx is being used as a framework for the current GraphQl implementation)

File: guides/v2.3/graphql/develop/resolvers.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Syntax option | Description
 `MutationQueryOutput` | The mutation's result type, such as  `String`, `Int`, or a custom type, like `MyCustomOutput`
 `@resolver(class)` | The class of the resolver
 `@doc(description)` | Describes the purpose of the mutation
+`@deprecated(reason: "description")` | Use `@deprecated` to mark a query or a query param as deprecated
```
[the remark this answers is not in the document]
Comment (keharper) [reply guides/v2.3/graphql/develop/resolvers.md]: Also add the same information to a new section at the bottom of `create-graphqls-file.md`.

[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.3/graphql/develop/resolvers.md]: > Also add the same information to a new section at the bottom of create-graphqls-file.md.

**added**

@keharper , could you please review?

Thank you!

**Comment by keharper** [comment]
Thanks @atwixfirster

**Pull request merged**